docs/data/migration/migration-charts-v6/migration-charts-v6.md

```diff
@@ -23,7 +23,7 @@ In `package.json`, change the version of the charts package to `next`.
 
 ## Update `@mui/material` package
 
-To have the option of using the latest API from `@mui/material`, the package peer dependency version has been updated to `^5.15.0`.
+To have the option of using the latest API from `@mui/material`, the package peer dependency version has been updated to `^5.15.13`.
```

Comment:
Updated the peerDep to the latest release for https://github.com/mui/mui-x/issues/12248#issuecomment-1994562544 and the mention in the migration docs.

Reply:
Why is updating `@mui/material` necessary?

Reply:
That discussion on Slack is about the `@mui/system` bump, not `@mui/material`.
Do we have to do both?

Reply:
Fair point. 🤔 
I'm not sure, haven't looked into it.
But for https://github.com/mui/mui-x/issues/12248 to work we'd need the latest `@mui/material` release. 🤔

Reply:
If I understand https://github.com/mui/mui-x/issues/12248 correctly, the user will have to use `@mui/material@5.15.13` for this to work, but it doesn't require bumping the minimum supported version of `@mui/material`.
Is this correct?

Reply:
Yes, that is correct, but if we would add a demonstration of that usage in the doc it would become strange to not support it with the least allowed version. 🤔

Reply:
Gotcha.
This still doesn't justify the peer dependency bump IMO, especially in the data grid 🙂
An alternative option would be to add a warning next to the demo, similar to the one we have in https://mui.com/x/react-data-grid/export/#using-a-web-worker

Reply:
I do see your point. 👌 
Depends how we look at it.
I remember a discussion about bumping the peer dependency versions to the latest ones before the stable release. 🤔 
They have already been bumped https://github.com/mui/mui-x/pull/11692 once.
If others agree, we can keep the existing peer dep version and when needed add the suggested information next to the demo. 👌

Reply:
I found this change (https://github.com/mui/material-ui/pull/40946) released in `5.15.12`.
WDYT, would it make sense to bump the peer dep version for it? 🤔

Reply:
Yes, makes sense to me.